bedrock/mozorg/redirects.py

```diff
@@ -562,4 +562,9 @@ def decider(request, **kwargs):
     redirect(r"^/exp/opt-out/?$", "privacy.convert-opt-out"),
     # Issue 13211
     redirect(r"^/en/$", "/en-US/", permanent=True),
+    # Issue 13554
+    redirect(r"^about/leadership/senior-leadership/$", "/about/leadership#senior/"),
```

Comment:
I think the trailing slash should go before the hash on these:
```suggestion
    redirect(r"^about/leadership/senior-leadership/$", "/about/leadership/#senior"),
```